src/plots/box.jl

```diff
@@ -1,10 +1,10 @@
 #Array of Arrays: awkward as data structure, awkward code but works
-function box(data::Vector{Vector{T}};
+function box(data::AbstractVector{<:AbstractVector{<:Union{Missing, Int, AbstractFloat, Rational}}};
```

Comment:
Is it necessary to restrict the `Real` subtypes like this? `Union{Missing, Real}` should work, if that's the functionality you're after.

Reply:
Probably. Originally, I wasn't sure if Rational would work, but then it did, so I just slapped it into the Union and moved on. Want to submit a PR for every method that does this? :)

Reply:
Oh, now I remember. I mistook irrational for imaginary, so I wanted to remove that. But that's a mistake, so yes, I mean `Real`